code/datums/radiation_wave.dm

```diff
@@ -112,8 +114,14 @@
 			))
 		if(!can_contaminate || blacklisted[thing.type])
 			continue
-		if(prob(contamination_chance)) // Only stronk rads get to have little baby rads
-			if(SEND_SIGNAL(thing, COMSIG_ATOM_RAD_CONTAMINATING, strength) & COMPONENT_BLOCK_CONTAMINATION)
```

Comment:
You also lost this signal

Reply:
i didn't, i put it up in the for loop

Reply:
I missed that and I'm really not a fan of the placement there but I'll get to that when you handle the other review